### PR TITLE
Custom download url

### LINKED ISSUE #27: Allow users to override url used to install terraform
As an enginer, I would like to be able to provide the full download path for the terraform executable, so that I can ensure terraform is installed from my company's approved location

### PR DESCRIPTION
**Pull request by favoretti**
Custom URL support and readme update.

Fixes #27

### CODE REVIEW

File: TerraformInstaller/src/installer.ts [hunk 1]
```diff
@@ -26,7 +26,12 @@ export async function download(inputVersion: string): Promise<string>{
     var version = latestVersion != "" ? sanitizeVersion(latestVersion) : sanitizeVersion(inputVersion);
     var cachedToolPath = tools.findLocalTool(terraformToolName, version);
     if(!cachedToolPath){
-        var url = getDownloadUrl(version);
+        var url: string;
+        if(downloadUrl) {
```
Comment (charleszipp) [inline TerraformInstaller/src/installer.ts:30]: nit: can simplify with coalesce. also use const/let over var to indicate if the local should be mutable.
```javascript
const url = downloadUrl || getDownloadUrl(version)
```

File: TerraformInstaller/task.json [hunk 2]
```diff
@@ -26,6 +26,14 @@
       "defaultValue": "latest",
       "required": true,
       "helpMarkDown": "Specify the version of terraform that should be installed"
+    },
+    {
+      "name": "downloadUrl",
+      "type": "string",
+      "label": "Download URL",
+      "defaultValue": "",
```
Comment (charleszipp) [inline TerraformInstaller/task.json:34]: Would remove this if there is no default value.

File: TerraformInstaller/src/installer.ts (hunk 1 above)
Comment (favoretti) [reply TerraformInstaller/src/installer.ts]: Thanks, node/ts noob here, great to learn this :)

File: TerraformInstaller/task.json (hunk 2 above)
Comment (favoretti) [reply TerraformInstaller/task.json]: Removed.

**Pull request merged**